docs/headless/openapi-specification/openapi.yaml

```diff
@@ -2458,8 +2458,14 @@ components:
                     description: |
                       A TOTP secret that can be used to setup a new authenticator.
                     example: J4ZKKXTK7NOVU7EPUVY23LCDV4T2QZYM
+                  totp_url:
+                    type: string
+                    description: |
+                      URL-encoded QR code containing the TOTP secret and other metadata to be scanned by OTP clients.
```

Comment:
This description seems off -- it is not a QR code right, nor is it URL encoded. It's just a URL/URI , from which the QR code could be generated.

Reply:
Do you want to link to e.g. https://datatracker.ietf.org/doc/draft-linuxgemini-otpauth-uri/